model/autocomplete.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type UserAutocomplete struct {
+	In  []*User `json:"in"`
```

Comment:
Yeah, I'm going to split it up into multiple APIs. Much simpler and cleaner that way